src/parsing.jl

```diff
@@ -266,12 +262,13 @@ function parse_numeric_array(eltype::Type{T}, str::AbstractString) where T
         end
 
         arr = OffsetArray{T}(undef, ranges...)
+        el_iter = eachmatch(r"[^\}\{,]+", str[eq_ind+1:end])
     else
         arr = Array{T}(undef, array_size(str)...)
+        el_iter = eachmatch(r"[^\}\{,]+", str)
```

Comment:
Given that in the original code `eq_ind` would have been `0` then this is looking at one extra character from the start. Instead of doing this should you just set `eq_ind = 0` and leave the `eachmatch` where it was?

Reply:
@iamed2 would be good to have a test case for why the first character should be skipped

Reply:
It's just the first `{` I believe. Skipping the first index isn't necessary. Probably left over from when I implemented it using `split`.

Reply:
In that case we can probably always do: ` el_iter = eachmatch(r"[^\}\{,]+", str)` without `eq_ind`

Reply:
No, the offset branch has to skip everything before the equals sign.

Reply:
So what is the resolution here? Is what's written acceptable?

Reply:
Yes the code is fine as written